CRM/Utils/Hook.php

```diff
@@ -2491,4 +2491,14 @@ public static function postIPNProcess(&$IPNData) {
     );
   }
 
+
+  /**
+   * Extensions can define new formats for relative date filter "tokens".
+   * @param string $filter - the filter token, stored in civicrm_option_value
+   * @return mixed $dates - Should return an array with two elements, $dates['from']
+   *  and $dates['to'].  Or FALSE if hook isn't in use.
+   */
+  public static function relativeDate($filter) {
+    return self::singleton()->invoke(array('filter'), $filter, self::$_nullObject, self::$_nullObject, self::$_nullObject, self::$_nullObject, self::$_nullObject, 'civicrm_relativeDate');
```

Comment:
I think code standard judging from nearby hooks is to put closing 
);
on a new line, outdented once.